test/Nancy.Tests/Unit/DynamicDictionaryValueFixture.cs

```diff
@@ -987,6 +987,38 @@ public void Should_return_default_value_if_implicit_convert_fails_on_datetime()
         }
 
         [Fact]
+        public void Should_adjust_to_universal_time_when_globalizationconfiguration_datetimestyles_requires_it()
+        {
+            // Given
+            var expected = DateTime.Parse("2016-05-24T08:41:37.1661565");
+
+            var config = new GlobalizationConfiguration(new [] {"en-US"}, dateTimeStyles: DateTimeStyles.AdjustToUniversal);
+            var value = new DynamicDictionaryValue("2016-05-24T10:41:37.1661565+02:00", config);

```

Comment:
I think what @jskeet said makes sense. Instead of `DateTime.Parse`, use a ctor and specify universal explicitly, or add `Z` to the string, but that's probably less obvious.

I now realize that time zones won't be an issue here. We're basically just testing that a local timestamp, `2016-05-24T10:41:37.1661565+02:00`, will be adjusted to a universal timestamp, `2016-05-24T08:41:37.1661565Z`. This is fiiiine...